src/views/core/bodies/ListFeePaymentsModal.vue

```diff
@@ -48,14 +55,41 @@
 <script>
 export default {
   name: 'ListFeePaymentsModal',
-  props: ['member'],
+  props: [
+    'member',
+    'payments',
+    'canDelete',
+    'route',
+    'services',
+    'root'
+  ],
   data () {
     return {
 
     }
   },
   methods: {
+    askDeleteMemberPaymentFee (fee) {
+      const message = `Are you sure you want to <b>delete</b> ${fee.id} from ${this.member.user.first_name} ${this.member.user.last_name}? This action cannot be undone.`
 
+      this.$buefy.dialog.confirm({
+        title: 'Deleting a fee',
+        message,
+        confirmText: 'Delete fee',
+        type: 'is-danger',
+        hasIcon: true,
+        onConfirm: () => this.deleteFee(fee)
+      })
+    },
+    deleteFee (fee) {
+      this.axios.delete(this.services['core'] + '/bodies/' + this.route.params.id + '/payments/' + fee.id).then(() => {
+        this.root.showSuccess('Fee is deleted.')
+        const updatedPayments = [...this.member.payments]
+        const index = updatedPayments.findIndex(p => p.id === fee.id)
+        updatedPayments.splice(index, 1)
+        this.member.payments = updatedPayments
```

Comment:
We should refresh the table as well, to prevent this behaviour after deleting the last fee payment;
<img width="515" alt="chrome_2024-05-17_16 16 37" src="https://github.com/AEGEE/frontend/assets/13023439/88a4d07e-90ab-4d23-b3f8-9d5b71a359cf">